erpnext/controllers/status_updater.py

```
@@ -323,6 +357,21 @@ def validate_qty(self):
 
+	def fetch_items_with_pending_qty(self, args, item_field, items):
+		return frappe.db.sql(
+			"""select name,`{item_code}` as item_code, `{target_ref_field}`,
+			`{target_field}`, parenttype, parent from `tab{target_dt}`
+			where `{target_ref_field}` < `{target_field}`
+			and name in %(names)s and docstatus=1""".format(
+				item_code=item_field,
+				target_ref_field=args["target_ref_field"],
+				target_field=args["target_field"],
+				target_dt=args["target_dt"],
+			),
```

Ruff (0.13.3)

Possible SQL injection vector through string-based query construction

(S608)